core/README.md

```diff
@@ -40,6 +40,24 @@ The `@ionic/core` package can by used in simple HTML, or by vanilla JavaScript w
 * [@ionic/angular](https://www.npmjs.com/package/@ionic/angular)
 
 
+## Custom Elements Build (Experimental)
```

Comment:
Is there anything else to do beside calling `initialize()` and registering the components ?

I am trying this feature of 5.6 and have a few issues:
-The body is initially hidden[1],
- The modals, alert, ... would not render - I am using the `modalController` from `@ionic/core/components` and registering `ion-modal` (`IonModal`) + `ion-backdrop`,
- I have a `<ion-select>` that is not rendering (supposed to be a popover) 

[1] because of 
```
html:not(.hydrated) body {
    display: none;
}
```

Reply:
Thanks for your super fast answer.

I have added the class to my html for now.

I'll dig a little bit in the overlay issue trying to find the root cause and I'll create an issue with a repro.